src/test/kotlin/com/exactpro/th2/processor/cache/collector/event/TestCacheEvent.kt

```diff
@@ -72,18 +74,21 @@ class TestCacheEvent {
         .build()
 
     private fun compare(cacheEvent: Event) {
-        assert(cacheEvent.book == grpcEvent.book)
-        assert(cacheEvent.scope == grpcEvent.scope)
-        assert(cacheEvent.id == eventId.id)
-        assert(cacheEvent.eventName == grpcEvent.name)
-        assert(cacheEvent.eventType == grpcEvent.type)
-        assert(cacheEvent.successful == grpcEvent.isSuccess())
-        assert(cacheEvent.body == grpcEvent.body.toStringUtf8())
+        assertEquals(cacheEvent.book, grpcEvent.book)
```

Comment:
grpcEvent must be parameter of compare method.
construction of grpcEvent must be done insede test case